### ISSUE TITLE
Cannot find module 'node:path'

### ISSUE THREAD

**Issue by luchanan** [gregnb/filemanager-webpack-plugin#101]
when i up upgrade to v6.1.0

Error: Cannot find module 'node:path'                                               │
   │   Require stack:                                                                      │
   │   - D:\project\nuxt\jm_m_nuxt\node_modules\filemanager-webpack-plugin\dist\index.js

**Comment by sibiraj-s** [comment]
What is the node version you have?

**Comment by mwoz123** [comment]
same here. 
```
 node --version
v10.13.0
```

```
    "filemanager-webpack-plugin": "^6.1.1",
```

**Comment by mwoz123** [comment]
Downgrading to `5.0.0` fixed/workaround issue (6.0.0 also didn't work for me.)

**Comment by sibiraj-s** [comment]
Yeah. It is a breaking change, Nodejs 10 has reached EOL, FMP v6 requires `^12.20.0 || ^14.13.1 || >=16.0.0` since . 

You can stick to v5 till then I suppose.

**Comment by guillaume-sainthillier** [comment]
Hello, 
This is the same for me: 
```bash
node --version
v14.17.1
```

Output (truncated): 
```
[webpack-cli] Failed to load '/Users/guillaume/Sites/sainthillier.fr/webpack.config.js'
[webpack-cli] Error: Cannot find module 'node:path'
Require stack:
- /Users/me/Sites/website/node_modules/filemanager-webpack-plugin/dist/index.cjs
- /Users/me/Sites/website/webpack.config.js
- /Users/me/Sites/website/node_modules/webpack-cli/lib/webpack-cli.js
- /Users/me/Sites/website/node_modules/webpack-cli/lib/bootstrap.js
- /Users/me/Sites/website/node_modules/webpack-cli/bin/cli.js
- /Users/me/Sites/website/node_modules/webpack/bin/webpack.js
```

**Comment by sibiraj-s** [comment]
Found the issue.  `node:` protocol support for CJS was added only in v16. Will fix this shortly

**Comment by sibiraj-s** [comment]
Fixed in v6.1.2

**Comment by guillaume-sainthillier** [comment]
That was quick! Thank you :)
